x-pack/plugins/search_playground/public/components/view_code/view_code_flyout.tsx

```diff
@@ -62,6 +65,15 @@ export const ViewCodeFlyout: React.FC<ViewCodeFlyoutProps> = ({ onClose }) => {
     'lc-py': LANGCHAIN_PYTHON(formValues, CLIENT_STEP),
     'py-es-client': PY_LANG_CLIENT(formValues, CLIENT_STEP),
   };
+  const handleLanguageChange = (e: React.ChangeEvent<HTMLSelectElement>) => {
+    setSelectedLanguage(e.target.value);
+
+    usageTracker.click(`${AnalyticsEvents.viewCodeLanguageChange}_${e.target.value}`);
+  };
+
+  useEffect(() => {
+    usageTracker.load(AnalyticsEvents.viewCodeFlyoutOpened);
```

Comment:
send the viewCodeLanguageChange event here on load too, on the current default target value.